src/client/components/map-vote-dialog/dialog.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { PureComponent } from 'react';
+import injectSheet from 'react-jss';
+import Aux from 'react-aux';
+import {
+  Button,
+  RadioButton,
+  RadioButtonGroup,
+  Typography,
```

Comment:
There are actual dialog content components, there is Dialog.Header, Dialog.Content and Dialog.Actions which you can use